liveblog/liveblog
# [LBSD-2823] - Scheduled posts do not appear in specific time

### Purpose

This PR fixes an issue where scheduled posts do not appear at the specific time when embed is left in the background 

### What has changed

- Change the query when `fromDate` is present when fetching new updates to be greater than or equal

### Steps to test
1. Navigate to the Editor.
2. Schedule posts at different intervals. 
3. Open the Live preview on a different tab and observe the scheduled post appear as expected

Resolves: [LBSD-2823](https://sofab.atlassian.net/browse/LBSD-2823)

[LBSD-2823]: https://sofab.atlassian.net/browse/LBSD-2823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
